README.md

```diff
@@ -1,2 +1,53 @@
-# Awesome-ES6-Implementation
-Awesome page with ES6 modules
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Awsome Books
+
+> In this project, we design a web page that creates a list of books. You can add a new book and remove a book from the list. The list is saved dynamically on the local storage, so every time you reload the page, the data saved on the local storage is displayed on the page.
+
+
+## Built With
+
+- HTML, CSS and Javascript
+- Linters
+-ES6
+
+
+## Live Demo
+
+> Coming soon.
+
+
+## Getting Started
+
+> You can download the source code and run it locally. The Live Demo is going to be available proximately.
+
+
```

Comment:
- [OPTIONAL] It will be nice to add explicit steps to clone and run your project locally, as this will be of help to non-technical users. Please add descriptive steps from cloning your app to seeing the results displayed on the browser.. This includes `git clone URL` , `cd project_name`,  `npm install`, `npm start`, etc

- please refer to this [template](https://github.com/microverseinc/readme-template), you can copy and paste the content but remember to customize it to your project.